stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -92,7 +92,7 @@ spec:
           image: "{{ .Values.Master.Image }}:{{ .Values.Master.ImageTag }}"
           imagePullPolicy: "{{ .Values.Master.ImagePullPolicy }}"
           {{- if .Values.Master.UseSecurity }}
-          args: [ "--argumentsRealm.passwd.$(ADMIN_USER)=$(ADMIN_PASSWORD)",  "--argumentsRealm.roles.$(ADMIN_USER)=admin"]
+          args: [ "--argumentsRealm.passwd.$(ADMIN_USER)=$(ADMIN_PASSWORD)",  "--argumentsRealm.roles.$(ADMIN_USER)=admin", "-Djavax.net.ssl.trustStore=$JENKINS_HOME/.keystore/cacerts" ]
```

Comment:
If I don't provide the certs, does supplying this argument break anything? Should it be contained within an if block like below?

Reply:
I am pretty sure I tested that possibility and it worked fine, but I am not remembering for certain at the moment. Let me retest to verify and I will report back my findings.

Reply:
Awesome. Thanks for checking!